commit 9c42b9fe352ef3e720fd306ab7c11a877c4a65f3
Author: TogoFire <togofire@mailfence.com>
Date:   Fri Nov 29 17:00:52 2024 -0300

    hal: lahaina: Drop hw acc 7d1580bd-297f-4683-9239-e475b6d1d69f
    
    - E EffectsFactoryConfigLoader: Error querying effect 7d1580bd-297f-4683-9239-e475b6d1d69f on lib offload_bundle
    
    Change-Id: 47c7404ae48127daf98f0a2346aa3627e5d04f51
    Signed-off-by: TogoFire <togofire@mailfence.com>

diff --git a/configs/lahaina/audio_effects.conf b/configs/lahaina/audio_effects.conf
index 8b9189a0..b109af80 100644
--- a/configs/lahaina/audio_effects.conf
+++ b/configs/lahaina/audio_effects.conf
@@ -221,10 +221,6 @@ effects {
     library downmix
     uuid 93f04452-e4fe-41cc-91f9-e475b6d1d69f
   }
-  hw_acc {
-    library offload_bundle
-    uuid 7d1580bd-297f-4683-9239-e475b6d1d69f
-  }
   loudness_enhancer {
     library loudness_enhancer
     uuid fa415329-2034-4bea-b5dc-5b381c8d1e2c
diff --git a/configs/lahaina/audio_effects.xml b/configs/lahaina/audio_effects.xml
index 15adc92f..2008405a 100644
--- a/configs/lahaina/audio_effects.xml
+++ b/configs/lahaina/audio_effects.xml
@@ -77,7 +77,6 @@
             <libhw library="visualizer_hw" uuid="7a8044a0-1a71-11e3-a184-0002a5d5c51b"/>
         </effectProxy>
         <effect name="downmix" library="downmix" uuid="93f04452-e4fe-41cc-91f9-e475b6d1d69f"/>
-        <effect name="hw_acc" library="offload_bundle" uuid="7d1580bd-297f-4683-9239-e475b6d1d69f"/>
         <effect name="loudness_enhancer" library="loudness_enhancer" uuid="fa415329-2034-4bea-b5dc-5b381c8d1e2c"/>
         <effect name="dynamics_processing" library="dynamics_processing" uuid="e0e6539b-1781-7261-676f-6d7573696340"/>
         <effect name="aec" library="audio_pre_processing" uuid="0f8d0d2a-59e5-45fe-b6e4-248c8a799109"/>
